runtime/moonriver/src/lib.rs

```diff
@@ -841,13 +841,78 @@ impl pallet_proxy::Config for Runtime {
 	type AnnouncementDepositFactor = ConstU128<{ currency::deposit(0, 56) }>;
 }
 
+use frame_support::pallet_prelude::PhantomData;
+use pallet_migrations::{GetMigrations, Migration};
+
+// Migration to insert dummy code for Conviction, Referenda, Preimages
+pub struct InsertDummyCodeOpenGov<T>(PhantomData<T>);
```

Comment:
It can be more readable to move this migration code in a new rust file `migrations.rs` in moonriver runtime crate